vajram/extensions/sql/vajram-sql/README.md

```
@@ -0,0 +1,658 @@
+
+This also eliminates N+1 problems by construction: nested `List<Projection>` methods generate a
+single LEFT JOIN query, not a loop of separate queries.
+
+---
+
+## Concepts
+
+### Tables
+
+A table is a [
+`@ModelRoot`](../../../../../../../../krystal-common/src/main/java/com/flipkart/krystal/model/ModelRoot.java)
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside link text

(MD039, no-space-in-links)